tests/e2e/specs/add-edit-category.test.js

```diff
@@ -0,0 +1,176 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	visitAdminPage,
+	pressKeyWithModifier,
+	pressKeyTimes
+} from '@wordpress/e2e-test-utils'
+import { addQueryArgs } from '@wordpress/url';
+
+describe( 'Categories tests', () => {
+	const title = 'New Category';
+	const query = addQueryArgs( '', {
+		taxonomy: 'category',
+	} );
+
+	beforeEach( async () => {
+		/**
+		 * Delete all categories before anything
+		 * This is useful because after running more than one test
+		 * there could be existing categories
+		 */
+		await visitAdminPage( 'edit-tags.php', query );
+		await page.$( '#bulk-action-selector-top' );
+		await page.waitForSelector( '[id^=cb-select-all-]' );
+		await page.click( '[id^=cb-select-all-]' );
+		await page.select( '#bulk-action-selector-top', 'delete' );
+		await page.click( '#doaction' );
+
+		/**
+		 * Create a new category with the title 'New Category'
+		 */
```

Comment:
This part could be moved out of `beforeEach` and into its own function. It's a bit more cumbersome as we'll have to call that function at the beginning of most of the tests, but that way we can also test the default state with no categories added, checking that "Uncategorized" is always present. 
Also, we may want to add other tests for that default state or for other behaviours in the future, so it's best to keep the setup steps as minimal as possible 🙂

Reply:
Make sense 🤔